### PR TITLE
Add some searchmenu localization strings

### PR DESCRIPTION
**Pull request by dm1tz**
Refs: [#14022](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/en_gb/?q=%2314022&sort_by=-priority%2Cposition&checksum=), [#593](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/en_gb/?q=%23593&sort_by=-priority%2Cposition&checksum=), [#342](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/en_gb/?q=%23342&sort_by=-priority%2Cposition&checksum=), [20343](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/en_gb/?q=%2320343&sort_by=-priority%2Cposition&checksum=)

### CODE REVIEW

**Comment by icarok99** [comment]
You have to translate this and wait for the Kodi team to accept your translation.

https://kodi.weblate.cloud/projects/kodi-core/kodi-main/uk_ua/

**Comment by dm1tz** [comment]
> You have to translate this and wait for the Kodi team to accept your translation.
> 
> https://kodi.weblate.cloud/projects/kodi-core/kodi-main/uk_ua/

Those are already translated, links in the description are for en_gb. See [#342](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/uk_ua/?checksum=3f0853933e8e1fed&q=+source%3AMovies+&sort_by=-priority%2Cposition), [#20343](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/uk_ua/?q=20343&sort_by=-priority%2Cposition&checksum=), [#14022](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/uk_ua/?q=14022&sort_by=-priority%2Cposition&checksum=), [#593](https://kodi.weblate.cloud/translate/kodi-core/kodi-main/en_gb/?q=%23593&sort_by=-priority%2Cposition&checksum=)

File: 1080i/Includes_Search_Filter.xml [hunk 1]
```diff
@@ -644,23 +644,23 @@
             </focusedlayout>
             <content>
                 <item>
-                    <label>All items</label>
+                    <label>$LOCALIZE[593]</label>
```
Comment (jurialmunkey) [inline 1080i/Includes_Search_Filter.xml]: 593 is just "All" not "All items". It's the wrong label. 
It's used for setting buttons e.g. "Shuffle: All" "Lock settings: All" "In-progress: All"

There is an "All items" label in the skin 31382 for this purpose:
https://github.com/jurialmunkey/skin.arctic.fuse/blob/221d025260640c246633477111d85968cf831bed/language/resource.language.en_gb/strings.po#L1333-L1336

**Review by jurialmunkey — CHANGES_REQUESTED** [review]

File: 1080i/Includes_Search_Filter.xml (hunk 1 above)
Comment (dm1tz) [reply 1080i/Includes_Search_Filter.xml]: Thanks, fixed.

**Pull request merged**